File: src/main/java/com/developerphil/adbidea/action/GrantPermissionsAction.java
package com.developerphil.adbidea.action;

import com.developerphil.adbidea.adb.AdbFacade;
import com.intellij.openapi.actionSystem.AnActionEvent;
import com.intellij.openapi.project.Project;

/**
 * Created by Stephan Hagios on 28.07.17.
 */
public class GrantPermissionsAction extends AdbAction {
    @Override
    public void actionPerformed(AnActionEvent e, Project project) {
        AdbFacade.grantPermissions(project);
    }
}

File: src/main/kotlin/io/github/raghavsatyadev/adbidea/action/GrantPermissionsAction.kt
package io.github.raghavsatyadev.adbidea.action

import com.intellij.openapi.actionSystem.AnActionEvent
import com.intellij.openapi.project.Project
import io.github.raghavsatyadev.adbidea.adb.AdbFacade

class GrantPermissionsAction : AdbAction() {
    override fun actionPerformed(e: AnActionEvent, project: Project) =
        AdbFacade.grantPermissions(project)
}
